tests/unit/worker/steps/test_dataset_pseudonymizer.py

```diff
@@ -1,21 +1,46 @@
-import json
 import os
-from pathlib import Path
+import json
 import shutil
+from pathlib import Path
+
 import pytest
+import pyarrow
+from pyarrow import parquet, dataset
 
 from job_executor.exception import BuilderStepError
 from job_executor.model import Metadata
 from job_executor.worker.steps import dataset_pseudonymizer
 from job_executor.adapter import pseudonym_service
 
 
+TABLE_SIZE = 1000
+INPUT_TABLE = pyarrow.Table.from_pydict(
+    {
+        "unit_id": [f"i{count}" for count in range(TABLE_SIZE)],
+        "value": ["a"] * TABLE_SIZE,
+        "start_year": [2020] * TABLE_SIZE,
```

Comment:
The existence of "start_year" may mislead to the conclusion that start_year is relevant for pseudonymization.
We know that "start_year" is relevant for partitioning not pseudo.
The test works fine though, maybe "start_year" should be removed for more clarity (?)

Reply:
Yes it since it only exists in some of the dataset I agree it just add noise. I'll remove it!